### DISCUSSION TITLE
Not able to use the markdown. Having error `Create node: Type listitem in node ListItemNode does not match registered node ListItemNode with the same type`

### DISCUSSION THREAD

**Discussion by venkateshpullaganti** [facebook/lexical#2992] [Q&A] [answered]
Hi, thank you for the excellent editor.
But I stumbled on a problem while rendering the markdown.
I have followed the below things to add the markdown to the lexical editor.
- Add nodes to the editor config 
- Render the `MarkdownShortcutPlugin`

Got below error:
![image](https://user-images.githubusercontent.com/38013987/189590930-5f348a0d-a464-4985-9323-f6759dd789a9.png)

I also checked the playground example. By following it is able to render some types of elements but not all like list. Any reference on implementing the markdown is much appreciated.
Also I am happy to open a PR for this example. :smile: 
TIA :smiley:

**Comment by venkateshpullaganti** [comment, marked as answer]
I found the answer. This is because of two reasons
- I am using different versions of the editor and nodes
- I am not providing the node to the list of supported nodes in the lexical editor initial config. 

If you found the same issue do check the above points. 
Thank you.

**Reply by vadimyen** [reply]
What do you mean by different versions of the editor and nodes? How is it possible? I have the same problem with CodeBlockNode.

**Reply by ahampriyanshu** [reply]
@vadimyen Check your `package.json` file.
